server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -69,8 +69,8 @@ public void testCreateTable() throws Exception {
                                  "\"primary_keys\":[\"col1\"]}," +
                                  "\"properties\":{" +
                                  // doc_values: true is default and not included
-                                 "\"col1\":{\"type\":\"integer\",\"position\":1}," +
-                                 "\"col2\":{\"type\":\"keyword\",\"position\":2}" +
+                                 "\"col1\":{\"type\":\"integer\",\"position\":1,\"oid\":2}," +
+                                 "\"col2\":{\"type\":\"keyword\",\"position\":2,\"oid\":1}" +
```

Comment:
OID-s are not necessarily aligned with pos - since `assignOidsToColumns` just iterates over metadata Map keys. 

 I don't think we need sorting by pos given that OID is purely internal thing and just "unique throughout the all history name". We will replace it by names when selecting _raw so users will not see oids.
 
Trying to sort here by positions would just put unnecessary burden on create/alter and more importantly dynamic updates.